include/io/mman-win32/mman.h

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
I'm actually not sure what to do here. Google code is shutting down, so that will be a dead link soon (year or so timeline). We could maintain a mirror under the meta-toolkit organization and credit the original source from there. It doesn't seem like the original author migrated anything as far as I can tell...